src/scss/components/_text-fields.scss

```diff
@@ -35,6 +35,16 @@
       width: 100%;
     }
   }
+
+  &.full-width {
+    width: 100%;
+    .md-text-field {
```

Comment:
Lines 41-46 can be merged with lines 91-93 which will also solve the scss lint error